openpower/package/ppe42-gcc/Config.in

```diff
@@ -1,7 +1,7 @@
 config BR2_PACKAGE_HOST_PPE42_GCC
         bool "ppe42-gcc"
-        default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_POWER9)
```

Comment:
also why?

Reply:
Same answer as the comment for openpower/package/ppe42-binutils/Config.in